### DISCUSSION TITLE
Background color for editor tabs

### DISCUSSION THREAD

**Discussion by cemccccc** [dbeaver/dbeaver#15935] [General]
Is there any way to change the color of an active tab in dbeaver? 

![image](https://user-images.githubusercontent.com/83164656/155934465-641aeb1d-14d4-47fd-a421-4569982fa462.png)

**Comment by Skydiveroid** [comment]
Could you please try resetting the workspace? You can find the instructions here: 
https://dbeaver.com/docs/wiki/Reset-workspace/

**Comment by Manja-S** [comment]
I did follow all of the instructions, including removing the .metadata folder but the color of the active tab didn't change for me.
(version 22.1.3)
